WordPress/Classes/ViewRelated/Comments/CommentsViewController+NetworkAware.swift

```diff
@@ -0,0 +1,26 @@
+import UIKit
+
+
+// CommentsViewController is an Objective-C class, so in order for interop to work, it looks like we need to override these methods.
+extension CommentsViewController: NetworkAwareUI {
+    func contentIsEmpty() -> Bool {
+        return tableViewHandler.resultsController.isEmpty()
+    }
+
+    @objc
```

Comment:
We usually put `@objc` on the same line as the function, any reason to do it differently here?

Reply:
I find it easier to read that way. I was not aware of that convention, updated!